curves/src/bls12_377/g2.rs

```diff
@@ -89,6 +104,13 @@ impl ShortWeierstrassParameters for Bls12_377G2Parameters {
     ) -> bool {
         p.mul_bits(BitIteratorBE::new(Self::ScalarField::characteristic())).is_zero()
     }
+
+    fn glv_endomorphism(
```

Comment:
Something that we might want to follow up on is that for G2 there exists an even more efficient decomposition relying on the GLS endomorphism (https://eprint.iacr.org/2008/194)

Reply:
Would you know the speedup this would give? If it's big I would consider doing it for this PR, otherwise might just want to create an issue.

Reply:
Let's create an issue, since it's not really on the critical path for anything (there's only one G2 mul in Marlin verification)

Reply:
Done!